package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "symbol-mobile-wallet",
-  "version": "1.4.1",
-  "private": true,
+  "version": "1.4.2",
+  "private": false,
```

Comment:
why do we set it to false?
> If you set "private": true in your package.json, then npm will refuse to publish it. This is a way to prevent accidental publication of private repositories. 
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Reply:
It hasn't been a private repository for quite some time now. But if it is a trick to avoid publishing it to NPM, I agree we can leave it as `true`. 

Thanks, fixed.